run_route_scripts/results/multi_diff_results.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
```

Comment:
What does this script do?  Can we add a note at the top

Reply:
added description